src/test/java/com/daml/extensions/testing/PartiesAllocationTest.java

```diff
@@ -28,8 +31,12 @@ public void testPartiesAreAllocated() {
     sandbox.getPartyId(CHARLIE);
   }
 
-  @Test(expected = NullPointerException.class)
+  @Test
   public void notAllocatedPartyThrows() throws NullPointerException {
-    sandbox.getPartyId("notAllocatedParty");
```

Comment:
Did this nice annotation go away (null pointer exception)?

Reply:
https://www.baeldung.com/junit-5-migration#1-annotations

yep...